autofit/non_linear/paths/directory.py

```diff
@@ -206,6 +208,18 @@ def load_search_internal(self):
         with open_(filename, "rb") as f:
             return dill.load(f)
 
+    def remove_search_internal(self):
+        """
+        Remove the internal representation of a non-linear search.
+
+        This deletes the entire `search_internal` folder, including a .pickle / .dill file containing the interal
+        results and files with the timer values.
+
+        This folder can often have a large filesize, thus deleting it can reduce hard-disk use of the model-fit.
+        """
+        shutil.rmtree(self.search_internal_path)
+        os.rmdir(self.search_internal_path)

```

Comment:
rmtree removes the top level directory so os.rmdir will throw a FileNotFoundError